features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/recorder/SessionReplayCustomCallbacks.kt

```diff
@@ -0,0 +1,13 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.sessionreplay.internal.recorder
+
+import android.view.Window
+
+data class SessionReplayCustomCallbacks(
+    val getCurrentWindows: (() -> List<Window>)? = null
```

Comment:
minor: there is no need to use `get` prefix for `val` property, it can be just `val currentWindows`